src/sentry/tasks/post_process.py

```diff
@@ -708,6 +708,18 @@ def get_event_raise_exception() -> Event:
         if should_update_escalating_metrics(event, is_transaction_event):
             _update_escalating_metrics(event)
 
+        try:
+            if not event.project.flags.has_high_priority_alerts:
+                from sentry.tasks.check_new_issue_threshold_met import check_new_issue_threshold_met
+
+                check_new_issue_threshold_met.delay(event.project)
```

Comment:
I think the cache check should go here, otherwise we're basically firing a task on every event until the threshold is met.